### DISCUSSION TITLE
Seeking Manual Verification for GitHub Student Authentication

### DISCUSSION THREAD

**Discussion by m7ploa1ws** [community/community#111373] [GitHub Education]
### Hi everyone,

Hello GitHub Community,

I hope this message finds you well. I've been attempting to verify my student status for the GitHub Student Developer Pack, but unfortunately, my attempts have been unsuccessful. I have a school-issued document with an official seal that proves my student identity, but the automated verification process hasn't worked for me.

Here are the key points I'd like to express:

1. **Multiple Attempts**: I've uploaded my proof of student status several times, but it hasn't been accepted.
2. **Sole Proof**: This document is the only evidence I have of my student identity.
3. **Seeking Manual Verification**: I'm reaching out to explore the possibility of manual verification.

Could someone please guide me on how to proceed with manual verification? Any assistance would be greatly appreciated.

Thank you in advance!

**Comment by ettaboyle** [comment]
The team just updated the [Student Developer Pack Application FAQs & Common Rejection Reasons (Updated!)](https://github.com/orgs/community/discussions/111352). Please check out that post for guidance and suggestions on all things related to the Student Developer Pack.

We'll share your feedback on your experience (described above) with the Education team in case this is a product related issue.

Thank you!
